src/prebid.js

```diff
@@ -655,4 +698,16 @@ pbjs.enableSendAllBids = function () {
   pb_sendAllBids = true;
 };
 
+pbjs.disableSendAllBids = function () {
```

Comment:
You're right—I do not see a case where it would be useful to disable sending all bids. I've removed this public function in the latest pull requests, instead using a variable on `pbjs` so that we can clean up calls to `enableSendAllBids` between tests.